src/components/icon/icon.svelte

```diff
@@ -22,7 +22,7 @@
     // 404).
     if (isSSR) return
 
-    requestIdleCallback(() => {
+    window.requestIdleCallback?.(() => {
```

Comment:
Sneaking a few extra things into this PR. Can move into separate PRs if needed.

@fallaciousreasoning, `requestIdleCallback` is not supported in Safari. Wonder if we should let it go (above approach), or check if it exists, and use an alternative if not found.

Reply:
Booooo Safari. What a pain. Probably fine to just not do anything on Safari - this can just stay as a progressive enhancement.

Reply:
Ok, so leave as is above, yes?